app/components/bali/carousel/component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Bali
+  module Carousel
+    class Component < ApplicationViewComponent
+      renders_many :images
+
+      def initialize(selected_image: 0, **options)
+        @options = options
+        @options = prepend_class_name(@options, 'carousel-component glide')
+        @options = prepend_controller(@options, 'carousel')
+        @options = prepend_data_attribute(@options, 'carousel-index-value', selected_image)
```

Comment:
acabo de crear un helper para hacer esto mas facil.

```suggestion
        @options = prepend_values(@options, 'carousel', { index: selected_image })
```